### PR TITLE
Add palindrome

### PR DESCRIPTION
**Pull request by Amisha-Mohapatra**

### CODE REVIEW

File: strings/java/paindrome.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+public class GFG {
```
[the remark this answers is not in the document]
Comment (Amisha-Mohapatra) [reply strings/java/paindrome.java:1]: @ming-tsai  follow gfg for my coding practice and use that code editor.. that's why named it..do you want me to rename the class?

[the remark this answers is not in the document]
Comment (ming-tsai) [reply strings/java/paindrome.java:1]: May be you could use class name like `Algorithm`, or `DSA`

[the remark this answers is not in the document]
Comment (Amisha-Mohapatra) [reply strings/java/paindrome.java:1]: sure..

**Comment by Amisha-Mohapatra** [comment]
@ming-tsai is it fine now?

**Review by ming-tsai — APPROVED** [review]
Nice work

**Comment by Amisha-Mohapatra** [comment]
> Nice work

Thank you and I will keep this mistake in mind.

**Pull request merged**